README.md

```diff
@@ -51,7 +54,12 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-This work was done in collaborative teams  and you cannot find it anywhere else
+
+This is my original work and no where else you can find it. but I have used some public materials which you can find down
```

Comment:
* Like the previous reviewer mentioned kindly give appropriate credit to the author of the design which you have used for this project. [Refer to the project requirements here ](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Original%20design%20idea,of%20your%20project.)